samples/bluetooth/direction_finding_central/sysbuild/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if (NOT BOARD MATCHES "nrf5340/cpuapp")
```

Comment:
I see that there is also another approach to board names. Some samples use `NORMALIZED_BOARD_TARGET` instead of `BOARD`. I think single standardized way would be the best. It would ease searching for boars in future e.g. every where `nrf5340/cpuapp` or `nrf5340_apuapp`.